Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:argon
```

Comment:
This should be based on centos like addons-server. Also node should come from our own builds which are mirrored from nodesource.

See https://bugzilla.mozilla.org/show_bug.cgi?id=1226533 for info.